### PR TITLE
added link to questions on home page

### LINKED ISSUE #2188: Add link to questions below intro paragraph on front page
@ebarry @steviepubliclab if this sounds good, I'd like to propose a simple link to the Questions page on the front page (http://publiclab.org when not logged in, or http://publiclab.org/home when you are), like this:

screenshot 2018-01-31 at 4 52 54 pm

What do you think? (from [this mockup](https://docs.google.com/presentation/d/1TCZoTfuhamRVrUak8aDgqJAwSgyhRtZg2Pgacl2_4zc/edit#slide=id.g2e47a5f7b9_0_0))

This could go just below our banners.

### PR DESCRIPTION
**Pull request by ViditChitkara**
closes #2188

Make sure these boxes are checked before your pull request is ready to be reviewed and merged. Thanks!

* [X] all tests pass -- `rake test:all`
* [X] code is in uniquely-named feature branch, and has been rebased on top of latest master (especially if you've been asked to make additional changes)
* [X] pull request is descriptively named with #number reference back to original issue

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/wiki/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. Please alert developers on plots-dev@googlegroups.com when your request is ready or if you need assistance.

Thanks!

### CODE REVIEW

**Comment by ViditChitkara** [comment]
home-page-1-1
How it looks now.

File: app/views/home/home.html.erb [hunk 1]
```diff
@@ -83,6 +92,34 @@
   .activity-dropdown {
     display: none;
   }
+
+  #questions-link-container {
+    padding: 94px 280px;
+    text-align: center;
+  }
+
+  #questions-link-container>h1 {
```
Comment (jywarren) [inline app/views/home/home.html.erb]: This is looking good, but I'm not sure we absolutely need the extra formatting here. I wonder if we could remove the styles below this line... and maybe use `<hr />` for spacing? I guess I'm reluctant to build out a big style section just for this one feature. What do you think?

**Comment by jywarren** [comment]
sounds great. Let's keep it minimal, using as much native Bootstrap styling
as we can!

**Comment by ViditChitkara** [comment]
Made the required changes. @jywarren please have a look.

**Comment by jywarren** [comment]
Awesome, would you mind posting a screenshot quickly? Thanks!!!

**Comment by ViditChitkara** [comment]
Hi, sorry for the late reply!!! Here's the screenshot
link-q-1

**Comment by jywarren** [comment]
Merged! Thanks so much. We hope to soon add the `/tags` page to this row as well!

**Comment by ViditChitkara** [comment]
Cool!!

**Pull request merged**